diff --git a/src/applications/phid/PhabricatorObjectHandle.php b/src/applications/phid/PhabricatorObjectHandle.php
index 1e6812b..6379915 100644
--- a/src/applications/phid/PhabricatorObjectHandle.php
+++ b/src/applications/phid/PhabricatorObjectHandle.php
@@ -417,7 +417,7 @@ final class PhabricatorObjectHandle
   public function getLinkName() {
     switch ($this->getType()) {
       case PhabricatorPeopleUserPHIDType::TYPECONST:
-        $name = $this->getName();
+        $name = $this->getFullName();
         break;
       default:
         $name = $this->getFullName();
